tez-dag/src/test/java/org/apache/tez/dag/app/dag/impl/TestVertexManager.java

```diff
@@ -24,14 +24,8 @@
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
-import static org.mockito.Matchers.any;
-import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
```

Comment:
please remove wildcard import, we usually refrain from doing that
by removing it we can stay consistent with other parts of the code

Reply:
Cool, OK, consistency is also important for me.
 But the tests don't always do that, namely the `any()` and the other matchers can be imported both as `org.mockito.Mockito.any` and as `org.mockito.ArgumentMatchers.any`. I would keep only one of these. The `org.mockito.Mockito.any` seems to be the better choice.

Reply:
this looks confusing from mockito side, can you please clarify the difference between them?

Reply:
These are the same. Basically the same code (function) is added to both the ArgumentMatchers and the Mockito classes. So in theory there is no real difference.